src/ec2017/ga/general/Population.java

```diff
@@ -86,8 +86,8 @@ public void evolve()
 				Individual parentA = matingPool.get(i);
 				Individual parentB = shuffledMatingPool.get(i);
 				
-				Individual child = parentA.crossOver(parentB, _crossOverOp);
-				newGeneration.add(child);
+				ArrayList<Individual> child = parentA.crossOver(parentB, _crossOverOp);
+				newGeneration.add((Individual) child);
```

Comment:
You can't cast an `ArrayList` to an `Individual`. You need to pull all the `Individual`s out of the list and add them to `newGeneration`.